src/layout/index.jsx

```diff
@@ -9,15 +9,23 @@ import "bootstrap/dist/css/bootstrap.min.css";
 import "./css/style.scss";
 import favicon from "../../static/logos/logo.png";
 
+
 // eslint-disable-next-line no-unused-vars
 export default ({ children, location }) => (
-  <body>
+  <React.Fragment>
     <Helmet>
       <meta name="description" content={config.siteDescription} />
       <link rel="icon shortcut" href={favicon} type="image/png" />
+      <meta property="og:title" content={config.siteTitle} />
+      <meta property="og:type" content="website" />
+      <meta property="og:url" content={config.siteUrl} />
+      <meta property="og:image" content={favicon} />
```

Comment:
Favicon? The image property is used to generate the thumbnail that appears on the shared link's preview fetched by the webcrawlers. So, either use some image from our current website "terasology.org" or keep it blank as of now.

Reply:
Alright. I think I will leave it blank for now.